pkg/crc/network/nameservers.go

```diff
@@ -100,3 +101,47 @@ func HasNameserverConfiguredLocally(nameserver NameServer) (bool, error) {
 
 	return strings.Contains(string(file), nameserver.IPAddress), nil
 }
+
+func GetResolvValuesFromHost() (*ResolvFileValues, error) {
```

Comment:
Why do you not refactor `func GetResolvValuesFromInstance` to reuse the same code?
This is unnecessary duplication as you could create the previous function to accept the multiline as input and return the filled struct.